### PR TITLE
Fixed crash when unknown packet arrives that test cannot parse

### PR DESCRIPTION
**Pull request by gshimansky**
Fix for #355
Also fixed MAC hardcoded address to be single cast.

### CODE REVIEW

File: test/stability/testCksum/testCksum.go [hunk 1]
```diff
@@ -423,7 +423,7 @@ func generatePacket(emptyPacket *packet.Packet, context flow.UserContext) {
 func initPacketCommon(emptyPacket *packet.Packet, length uint16, rnd *rand.Rand) {
 	// Initialize ethernet addresses
 	emptyPacket.Ether.DAddr = [6]uint8{0xde, 0xea, 0xad, 0xbe, 0xee, 0xef}
-	emptyPacket.Ether.SAddr = [6]uint8{0x11, 0x22, 0x33, 0x44, 0x55, 0x66}
+	emptyPacket.Ether.SAddr = [6]uint8{0x10, 0x22, 0x33, 0x44, 0x55, 0x66}
```
Comment (ifilippov) [inline test/stability/testCksum/testCksum.go]: Comment here

File: test/stability/testSingleWorkingFF/testSingleWorkingFF.go [hunk 2]
```diff
@@ -446,22 +450,23 @@ func generatePacket(pkt *packet.Packet, context flow.UserContext) {
...
 	if commonCheck(pkt) == false {
-		return
+		return false
 	}
 	udp := pkt.GetUDPForIPv4()
 	if udp.DstPort != packet.SwapBytesUint16(dstPort1) {
 		println("Unexpected packet in inputFlow1", udp.DstPort)
 		atomic.AddUint64(&brokenPackets, 1)
-		return
+		return false
```
Comment (ifilippov) [inline test/stability/testSingleWorkingFF/testSingleWorkingFF.go]: No, you can't use this function in Receive part. In Receive part you will have all packets at receive. It means that there will be packets with "udp.DstPort != packet.SwapBytesUint16(dstPort1)". Can you please run tests after these changes?

Comment (gshimansky) [reply test/stability/testSingleWorkingFF/testSingleWorkingFF.go]: Yes you are right. I changed receive side to call ShouldBeSkipped function instead.

**Comment by ifilippov** [comment]
Approved

**Pull request merged**